.gitpod.yml

```diff
@@ -4,7 +4,19 @@ image:
 checkoutLocation: dazzle
 workspaceLocation: dazzle
 tasks:
+  - command: sudo buildkitd --group gitpod
+  - command: docker run -p 5000:5000 registry
+    openMode: split-right
   - init: go get -v ./...
+    openMode: tab-after
+    command: |
+      gp await-port 5000
+      go generate -v ./...
+      go run main.go build --context example $(gp url 5000 | cut -d/ -f3)/dazzle
```

Comment:
I'd prefer to see this in the Makefile so it is a little more standard & more easily reused.
See [my branch](https://github.com/gitpod-io/dazzle/blob/rl/upgrade-all-the-things/example/Makefile)

Reply:
#13 removes the Makefile :)

Reply:
Hrm, I've been expanding the Makefile... go + Makefiles are a common expectation.